src/transaction/components/TransactionToast.tsx

```diff
@@ -1,13 +1,12 @@
-import { useCallback, useEffect, useMemo } from 'react';
-import { closeSvg } from '../../internal/svg/closeSvg';
-import { background, cn } from '../../styles/theme';
+import { useCallback } from 'react';
+import { Toast } from '../../internal/components/Toast';
 import type { TransactionToastReact } from '../types';
 import { useTransactionContext } from './TransactionProvider';
 
 export function TransactionToast({
   children,
   className,
-  durationMs = 3000,
+  durationMs = 8000,
```

Comment:
shouldn't these still be 3000?

Reply:
That's what we currently have, but in my testing, 3 seconds seems really short. I'm barely able to read the toast and move my cursor over before it disappears.

Reply:
@0xAlec thoughts?

Reply:
reverting to 5000ms